CRM/CloseAccountingPeriod/BAO/CloseAccountingPeriod.php

```diff
@@ -195,8 +195,8 @@ public static function exportTrialBalanceAndClosePeriod($orgId) {
     $rows[] = array(
       NULL,
       NULL,
-      CRM_Utils_Money::format($debit),
-      CRM_Utils_Money::format($credit),
+      CRM_Utils_Money::formatLocaleNumericRoundedByOptionalPrecision($debit, 9),
```

Comment:
So these won't return the same value, in the original it will include the currency in the latter it won't be returning the currency